tensorrt_llm/_torch/custom_ops/flashinfer_custom_ops.py

```
@@ -63,6 +63,30 @@ def flashinfer_fused_add_rmsnorm(input: torch.Tensor,
                           eps,
                           enable_pdl=get_env_enable_pdl())
 
+    @torch.library.custom_op("trtllm::flashinfer_fused_add_add_rmsnorm",
+                             mutates_args=("input", "residual"))
+    def flashinfer_fused_add_add_rmsnorm(input: torch.Tensor,
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)